x-pack/plugins/security_solution/public/management/components/console/hooks/state_selectors/use_data_test_subj.ts

```diff
@@ -7,6 +7,16 @@
 
 import { useConsoleStore } from '../../components/console_state/console_state';
 
-export const useDataTestSubj = (): string | undefined => {
-  return useConsoleStore().state.dataTestSubj;
+/**
+ * Returns the `data-test-subj` that was defined when the `Console` was rendered.
+ * Can optionally set a suffix on that value if one is provided
+ */
+export const useDataTestSubj = (suffix: string = ''): string | undefined => {
```

Comment:
The return type should be just `string`. No?

Reply:
ahh. yes, you right. Will correct it.